docs/cn-demos.md

```diff
@@ -11,7 +11,7 @@ Before running the demo setup script, ensure you have the following prerequisite
 
 * **Python:** Python versionn >=3.9
 * **uv** [uv](https://docs.astral.sh/uv/getting-started/installation) must be installed (e.g. `pip install uv`)
-* **Docker:** Docker Desktop, Rancher Desktop or Podman Machine.
+* **Docker:** Docker Desktop, Rancher Desktop or Podman Machine. On MacOS, you will need also to do `brew install docker-credential-helper`
```

Comment:
nice addition! I mentioned that later in the Troubleshooting for Podman, but it should be done here too